### PR TITLE
Fix length of separators

### PR DESCRIPTION
**Pull request by masterleinad**
Some of the source code comments we use to indicate the purpose of the following lines were split into multiple lines. I searched for all consecutive lines having three dashes at the begin and end.

### CODE REVIEW

File: include/deal.II/base/bounding_box.h [hunk 1]
```diff
@@ -170,10 +169,9 @@ inline BoundingBox<spacedim, Number>::BoundingBox(
 {
   // We check the Bounding Box is not degenerate
   for (unsigned int i = 0; i < spacedim; ++i)
-    Assert(
-      boundary_points.first[i] <= boundary_points.second[i],
-      ExcMessage(
-        "Bounding Box can't be created: the point's order should be bottom left, top right!"));
+    Assert(boundary_points.first[i] <= boundary_points.second[i],
+           ExcMessage("Bounding Box can't be created: the point's "
+                      "order should be bottom left, top right!"));
```
Comment (masterleinad) [inline include/deal.II/base/bounding_box.h]: This I fixed additionally manually.

Comment (kronbichler) [reply include/deal.II/base/bounding_box.h]: A nit-pick: Shouldn't this be "the points' order"?

Comment (masterleinad) [reply include/deal.II/base/bounding_box.h]: > A nit-pick: Shouldn't this be "the points' order"?

Sure. :smiley:

Comment (bangerth) [reply include/deal.II/base/bounding_box.h]: I've taken to just editing people's patches for these obvious things. The github editor is good enough for that.

**Pull request merged**